bindings/matrix-sdk-ffi/Cargo.toml

```
@@ -65,6 +65,7 @@ features = [
     "e2e-encryption",
     "experimental-oidc",
     "experimental-sliding-sync",
+    "experimental-widget-api",
```

```suggestion
    "experimental-widget",
```

We don't have `oidc-api`, no `sliding-sync-api`, so let's keep it simple with `widget` :-).